src/components/autocomplete/autocomplete.scss

```diff
@@ -110,9 +110,18 @@ md-autocomplete {
     padding: 0 15px;
     line-height: 40px;
     height: 40px;
+    position: absolute;
+  }
+  // Dimensions of md-autocomplete-placeholder should match md-autocomplete.
+  .md-autocomplete-placeholder {
+    color: #757575;
```

Comment:
The color for autocomplete needs to be defined as part of the theme, otherwise this will be off for dark themes.